### ISSUE TITLE
Server crash when entering dbstats command in server console on sqlite

### ISSUE THREAD

**Issue by PTR-inc** [Ylianst/MeshCentral#4430]
Don't know if this is exclusive for the sqlite db, but when issueing the dbstats command on the server console, the server crashes

**To Reproduce**
1. Go to 'My Server/Console'
2. Enter 'dbstats'

**Expected behavior**
See the dbstats

**Server Software (please complete the following information):**
 - OS: Ubuntu 22.04.1
 - Virtualization: No
 - Network: WAN
 - Version: 1.0.71
 - Node: 18.7.0

**Additional context**
dbcounters command works

**servererror**
```
-------- 8/22/2022, 9:34:17 AM ---- 1.0.71 --------

/opt/meshcentral/node_modules/meshcentral/db.js:1508
                sqlDbQuery('SELECT COUNT(*) FROM main', null, function (err, response, raw) { obj.stats.meshcentral = (err == null ? parseInt(raw.rows[0].count) : 0); if (--obj.stats.c == 0) { delete obj.stats.c; func(obj.stats); } });
                                                                                                                                                  ^

TypeError: Cannot read properties of undefined (reading 'rows')
    at /opt/meshcentral/node_modules/meshcentral/db.js:1508:147
    at Statement.<anonymous> (/opt/meshcentral/node_modules/meshcentral/db.js:1207:29)

Node.js v18.7.0
```

**config**
```
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the name. Ref>  "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
  "settings": {
    "cert": "****",
    "WANonly": true,
    "_LANonly": true,
    "_sessionKey": "MyReallySecretPassword1",
    "port": 443,
    "_aliasPort": 443,
    "redirPort": 80,
    "_redirAliasPort": 80,
    "webRTC": true,
    "ClickOnce": true,
    "AllowLoginToken": false,
    "desktopMultiplexor" : true,
    "noAgentUpdate": true,
    "cleanNpmCacheOnUpdate": false,
    "browserPing": 60,
    "browserPong": 60,
    "agentPing": 60,
    "agentPong": 60,
    "agentIdleTimeout": 64,
    "SQLite3": true
  },
  "domains": {
    "": {
      "title": "****",
      "title2": "****",
      "footer": "***",
      "mstsc": true,
      "minify": true,
      "SiteStyle": 2,
      "ssh": true,
      "MeshQuota": 10000000,
      "UserQuota": 1000000,
      "newAccounts": 0,
      "_userNameIsEmail": true,
      "PasswordRequirements": { "min": 10, "max": 32, "upper": 0, "lower": 0, "numeric": 0, "nonalpha": 0},
      "myServer": {
        "Upgrade": false,
        "Errorlog": true,
        "Console": true,
        "Trace": true,
        "Backup": true,
        "Restore": true
      }
    }
  },
  "letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "***",
    "names": "***,
    "production": true
  }
}
```

**Comment by Ylianst** [comment]
Thanks for reporting that, I will fix it.

**Comment by Ylianst** [comment]
I just put in a fix for this, will be in v1.0.72.
